pkg/printers/internalversion/printers.go

```diff
@@ -2779,6 +2787,41 @@ func printClusterCIDRList(list *networking.ClusterCIDRList, options printers.Gen
 	return rows, nil
 }
 
+func printIPAddress(obj *networking.IPAddress, options printers.GenerateOptions) ([]metav1.TableRow, error) {
+	row := metav1.TableRow{
+		Object: runtime.RawExtension{Object: obj},
+	}
+
+	parentRefName := "<none>"
+	if obj.Spec.ParentRef != nil {
```

Comment:
Validation does not allow a nil value.

Reply:
yeah, I like your thoughts on this
I started with ParentRef as optional, but then I thought that  IPAddress is an object to be owned always and for different objects, Services, GatewayAPI, ServiceMesh ...  and allowing to leave dangling IPAddresses will make it hard for implementing multiple controllers ... who reconciles what, and garbage collect, ... having always a reference makes this much simpler

Reply:
I just keep this in case we revert the original decisions and ParentRef moves to optional, this is a printer and will not have any other effect